tests/models/wav2vec2_conformer/test_modeling_wav2vec2_conformer.py

```diff
@@ -872,6 +919,57 @@ def _load_datasamples(self, num_samples):
 
         return [x["array"] for x in speech_samples]
 
+    def check_inference_pretrained(self, model_id, model, input_name, inputs, features_shape):
```

Comment:
(nit) I believe `features_shape` can be removed as an argument since it's defined as:
```python
features_shape = inputs[input_name][:2]
```
for both fbank and raw audio models